diff --git a/optimizer.cjs b/optimizer.cjs
index add5bb6089e10e3c41de62ce9cc012aec5ded67f..32dfc0dbdcb21a85539602b447117ffa50475ce2 100644
--- a/optimizer.cjs
+++ b/optimizer.cjs
@@ -2978,7 +2978,7 @@ globalThis.qwikOptimizer = function(module) {
   var VALID_ID_PREFIX = "/@id/";
   var VITE_PUBLIC_PATH = "/@vite/";
   var internalPrefixes = [ FS_PREFIX, VALID_ID_PREFIX, VITE_PUBLIC_PATH ];
-  var InternalPrefixRE = new RegExp(`^(?:${internalPrefixes.join("|")})`);
+  var InternalPrefixRE = new RegExp(`(?:${internalPrefixes.join("|")})`);
   var shouldSsrRender = (req, url) => {
     const pathname = url.pathname;
     if (/\.[\w?=&]+$/.test(pathname) && !pathname.endsWith(".html")) {
diff --git a/optimizer.mjs b/optimizer.mjs
index 49de5cc48e365a2935e98aedbb17a8eb26fb570c..8ffbbd31913860ec0762cc6d87cb3f91083e6865 100644
--- a/optimizer.mjs
+++ b/optimizer.mjs
@@ -3086,7 +3086,7 @@ var VITE_PUBLIC_PATH = "/@vite/";
 
 var internalPrefixes = [ FS_PREFIX, VALID_ID_PREFIX, VITE_PUBLIC_PATH ];
 
-var InternalPrefixRE = new RegExp(`^(?:${internalPrefixes.join("|")})`);
+var InternalPrefixRE = new RegExp(`(?:${internalPrefixes.join("|")})`);
 
 var shouldSsrRender = (req, url) => {
   const pathname = url.pathname;